sys/targets/targets.go

```diff
@@ -677,6 +678,13 @@ func initTarget(target *Target, OS, arch string) {
 		// https://github.com/google/syzkaller/commit/06db3cec94c54e1cf720cdd5db72761514569d56
 		target.Triple = ""
 	}
+	target.ExternalModConf = ""
+	if OS == Linux {
+		kconfExtPref, found := os.LookupEnv("KCONFIG_EXT_PREFIX")
```

Comment:
What's the meaning of this variable? How are users supposed to set it? How do you set it?
It's still coming from env (non-hermetic) and nobody will set it.

Reply:
> What's the meaning of this variable? How are users supposed to set it? How do you set it? It's still coming from env (non-hermetic) and nobody will set it.

The variable has been added to Kernel config for Android13-5.15 to support external (vendor) modules.
https://android.googlesource.com/kernel/common/+/refs/heads/android13-5.15/Kconfig#34
See this commit for more info:
https://android.googlesource.com/kernel/common/+/9fe4dc55985d6b82ed749e2a6add7ed827ede35d
By default it should be set to an empty string, and in that case the build will expand it to the default, empty file:
https://android.googlesource.com/kernel/common/+/refs/heads/android13-5.15/Kconfig.ext
Yes, it is coming from an environment variable. The vendors will set it to whatever they will want to build into their kernels.
With this change you don't need to set anything, but if you do set it, then the 'make configs' will expand it and look for the Kconfig.ext in the path that you set in KCONFIG_EXT_PREFIX, relative to the $SOURCEDIR.

Reply:
... and without this change the 'make configs' will fail with:
```
android-5.15 failed:
/syzkaller/kernel/Kconfig:35:41: open /syzkaller/kernel/$(KCONFIG_EXT_PREFIX)Kconfig.ext: no such file or directory
source "$(KCONFIG_EXT_PREFIX)Kconfig.ext"
```
for android13-5.15 and above.

Reply:
I tried to use empty env variable and it doesn't work.
How to specify "$(KCONFIG_EXT_PREFIX)" for the build w/o this patch?